lstchain/data/lstchain_standard_config.json

```diff
@@ -148,14 +151,6 @@
        "charge_median_cut_outliers": [-0.5,0.5],
        "charge_std_cut_outliers": [-10,10],
        "time_cut_outliers": [2,38]
-     },
```

Comment:
this is the configuration for the calibrator and should be left as it was

Reply:
ha yes I think I misread the config, reverted that